scripts/zones/Arrapago_Reef/npcs/_1ie.lua

```diff
@@ -0,0 +1,66 @@
+-----------------------------------
+-- Area: Arrapago Reef
+-- Door: Iron Gate (Lamian Fang Key)
+-- !pos 580 -17 120
+-----------------------------------
+package.loaded["scripts/zones/Arrapago_Reef/TextIDs"] = nil
+-----------------------------------
+require("scripts/zones/Arrapago_Reef/TextIDs")
+require("scripts/globals/status")
+
+function onTrade(player,npc,trade)
+    if npc:getAnimation() == ANIMATION_CLOSE_DOOR then
```

Comment:
if npc:getAnimation() == dsp.anim.CLOSE_DOOR (replace all the others too)

Reply:
yep, forgot we moving everything to tables... I'll have to fix it